damask_parse/readers.py

```diff
@@ -167,9 +167,9 @@ def read_geom(geom_path):
                 volume element.
 
     """
-    from damask import Grid
+    from damask import GeomGrid
```

Comment:
In this case, it's not too much bother to maintain compatibility across versions, with something like this:

```py
try:
    from damask import GeomGrid as grid_cls
except ImportError:
    from damask import Grid as grid_cls
```
and then, e.g.:

```py
ve_grid = grid_cls.load(geom_path)
```

I would prefer something like that!

Reply:
Great, thanks. I've updated the PR.